sky/templates/gcp-ray.yml.j2

```diff
@@ -54,6 +54,7 @@ available_node_types:
             diskSizeGb: {{disk_size}}
             # See https://cloud.google.com/deep-learning-vm/docs/images
             sourceImage: {{image_id}}
+            diskType: zones/{{zones}}/diskTypes/{{disk_type}}
```

Comment:
Need to confirm: It seems that I cannot stop the instance when the `disk_type` is specified as `pd-ssd`. Could you confirm if that is true? If that is the case we may need to think about how to get around that.